app/api/V2/__init__.py

```diff
@@ -7,7 +7,7 @@
 from .views.parcelviews import MovingParcels, SpecificParcel, DeliverParcel
 from .views.parcelviews import DeclinedParcels, MarkParcelInTransit
 from .views.parcelviews import ApproveParcel, GetAcceptedParcels, DeclineParcel
-from .views.parcelviews import UserRegistration
+from .views.userviews import UserRegistration
 
 version_2 = Blueprint('v2', __name__)
 
```

Comment:
there should be only one line space at the end of the files. Let this be the last time I'm repeating this feedback.